### PR TITLE
Fix up MSBuild API ref

### PR DESCRIPTION
**Pull request by gewarren**
Remarks tags were inserted into summary tags resulting in things like this:

image

Also improves other property summaries and values.

### CODE REVIEW

File: xml/Microsoft.Build.Framework/SdkResult.xml [hunk 1]
```diff
@@ -14,13 +14,17 @@
   </Base>
   <Interfaces />
   <Docs>
-    <summary>
-                An abstract interface class to indicate SDK resolver success or failure.
-                <remarks>
```
Comment (gewarren) [inline xml/Microsoft.Build.Framework/SdkResult.xml:19]: @joelmartinez It seems like a build warning could/should be generated if `<remarks>` tags are found within `<summary>` or `<returns>` tags?

Comment (joelmartinez) [reply xml/Microsoft.Build.Framework/SdkResult.xml:19]: @mimisasouvanh this sounds like a great customer suggestion :)

**Comment by gewarren** [comment]
Close/reopen.

**Comment by gewarren** [comment]
Ping @ghogen.

**Pull request merged**